### PR TITLE
adding mean function

### PR DESCRIPTION
**Pull request by senzhanopt**

### CODE REVIEW

File: mean.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def mean(a,b)
```
Comment (senzhanopt) [inline mean.txt:1]: add a ":"

File: mean.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+"""Calculates the mean of two values"""
+return (a+b)/2
```
Comment (senzhanopt) [inline mean.txt:3]: add four blank spaces

**Review by senzhanopt — COMMENTED** [review]
Thanks, but could you follow the requests?

File: mean.txt (hunk 1 above)
Comment (zbounik) [reply mean.txt:1]: OK, nice remark :)

**Review by senzhanopt — COMMENTED** [review]
Looks good, thanks

**Pull request merged**